api/Attr.json

```diff
@@ -214,6 +308,100 @@
             "deprecated": false
           }
         }
+      },
+      "specified": {
```

Comment:
This is a funny attribute, it always returns true :)

At some point I think the API was able to *create* attributes that didn't yet exist via an `Attr` node, but no more...